test.py

```diff
@@ -0,0 +1,52 @@
+from prem import PremAI
+
+api_key = "3EC9HA3EFaQgFbzE8TqsOb0OTrTfFxv2x9"
+base_url = "http://localhost:8000"
```

Comment:
We should have a default base_url pointing to prod

Reply:
what's the prod url?

Reply:
https://app.prod.prem.ninja/

Reply:
But we probably need to change it to premai.io

Reply:
done